specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2022-11-30-preview/definitions/MigrateMySqlAzureDbForMySqlSyncTask.json

```diff
@@ -0,0 +1,483 @@
+          "items": {
+            "$ref": "#/definitions/MigrateMySqlAzureDbForMySqlSyncDatabaseInput"
+          }
+        },
+        "makeSourceServerReadOnly": {
+          "type": "boolean",
+          "description": "Setting to set the source server read only",
+          "default": false
+        },
+        "startedOn": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Parameter to specify when the migration started"
+        },
+        "sourceServerResourceId": {
+          "type": "string",
+          "description": "Optional resource Id of the source server if it is an azure instance"
+        },
+        "targetServerResourceId": {
+          "type": "string",
+          "description": "Optional resource Id of the target server"
+        },
+        "optionalAgentSettings": {
+          "type": "object",
+          "description": "Optional parameters for fine tuning the data migration",
+          "properties": {
+            "enableConsistentBackup": {
+              "type": "boolean",
+              "description": "Setting to enable transactional consistency",
+              "default": false
+            }
+          },
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "migrateAllViews": {
+          "type": "boolean",
+          "description": "If true, all view definitions will be migrated in the selected databases",
+          "default": false
+        },
+        "migrateAllTriggers": {
+          "type": "boolean",
+          "description": "If true, all trigger definitions will be migrated in the selected databases",
+          "default": false
+        },
+        "migrateAllEvents": {
+          "type": "boolean",
+          "description": "If true, all event definitions will be migrated in the selected databases",
+          "default": false
+        },
+        "migrateAllRoutines": {
+          "type": "boolean",
+          "description": "If true, all routine definitions will be migrated in the selected databases",
+          "default": false
+        },
+        "migrateAllTablesSchema": {
+          "type": "boolean",
+          "description": "If true, all table's schemas will be migrated",
+          "default": false
+        },
+        "migrateUserSystemTables": {
+          "type": "boolean",
+          "description": "If true, all users/grants will be migrated",
+          "default": false
+        },
+        "encryptedKeyForSecureFields": {
+          "type": "string",
+          "description": "encrypted key for secure fields"
+        }
+      },
+      "required": [
+        "sourceConnectionInfo",
+        "targetConnectionInfo",
+        "selectedDatabases"
+      ]
+    },
+    "MigrateMySqlAzureDbForMySqlSyncTaskOutput": {
+      "type": "object",
+      "description": "Output for the task that migrates MySQL databases to Azure Database for MySQL for online migrations",
```

Comment:
Same for the descriptions in this file, should it follow the phrasing you used above in databseInput? So as to not use task that migrated ... for migration?

Reply:
This is an existing comment that was transferred from the previous version of the API. I agree that it is ugly, and I will change it.